tests/test_ui.py

```diff
@@ -0,0 +1,597 @@
+# -*- coding: utf-8 -*-
+
+"""UI tests."""
+
+import os
+import json
+
+import pytest
+import sphinx
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from tests.utils import InjectJsManager, set_viewport_size
+from tests import TEST_DOCS_SRC
+
+
+DUMMY_RESULTS = os.path.join(
+    os.path.abspath(os.path.dirname(__file__)),
+    'dummy_results.json'
+)
+
+READTHEDOCS_DATA = {
+    'project': 'docs',
+    'version': 'latest',
+    'language': 'en',
+    'api_host': 'https://readthedocs.org/',
+}
+
+# This will be inserted in the html page
+# to support the working of extension
+# in test cases.
+SCRIPT_TAG = '<script>var READTHEDOCS_DATA = {};</script>'.format(
+    json.dumps(READTHEDOCS_DATA)
+)
+
+
+def open_search_modal(driver):
+    """Open search modal and checks if its open correctly."""
+    sphinx_search_input = driver.find_element_by_css_selector(
+        'div[role="search"] input'
```

Comment:
Out of curiosity, what would happen if this element appeared twice on the page?

Reply:
It would take the first one -- https://selenium-python.readthedocs.io/locating-elements.html#locating-elements-by-css-selectors